### DISCUSSION TITLE
I'm using React Query v4: Do I also need to install v4 of the dev tools as well?

### DISCUSSION THREAD

**Discussion by Fullchee** [TanStack/query#6861] [Q&A] [answered]
If that's the case, I can update the docs and add the `@4` suffix to the docs!

https://tanstack.com/query/v4/docs/framework/react/devtools

**Comment by TkDodo** [comment, marked as answer]
yes, the major version of all the packages must be in sync

**Comment by Fullchee** [comment]
@TkDodo

Thank you!

I created this PR to update the docs!
- https://github.com/TanStack/query/pull/6941
